### PR TITLE
adds log-ios and log-android cli commands

### PR DESCRIPTION
**Pull request by vitalis**
solves first part of #8186

### CODE REVIEW

**Comment by facebook-github-bot-2** [comment]
By analyzing the blame information on this pull request, we identified @janicduplessis and @hramos to be potential reviewers.

**Comment by facebook-github-bot-2** [comment]
Thank you for your pull request and welcome to our community. We require contributors to sign our Contributor License Agreement, and we don't seem to have you on file. In order for us to review and merge your code, please sign up at https://code.facebook.com/cla - and if you have received this in error or have any questions, please drop us a line at cla@fb.com. Thanks!

File: local-cli/logAndroid/logAndroid.js [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+      throw log.error;
+    }
+
+  } catch (e) {
+    console.log(chalk.red(
+      'adb invocation failed. Do you have adb in your PATH?'
+    ));
+    // stderr is automatically piped from the gradle process, so the user
+    // should see the error already, there is no need to do
+    // `console.log(e.stderr)`
```
Comment (mkonicek) [inline local-cli/logAndroid/logAndroid.js]: You can probably remove these lines (copied from run-android, mentions gradle).

File: local-cli/logIOS/logIOS.js [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+function _logIOS(resolve, reject) {
+  let rawDevices;
+
+  try {
+    rawDevices = child_process.execFileSync(
+      'xcrun', ['simctl', 'list', 'devices', '--json'], {encoding: 'utf8'}
+    );
+  } catch (e) {
+    console.log(chalk.red(
+      'xcrun invocation failed. Please see that Xcode is installed.'
```
Comment (mkonicek) [inline local-cli/logIOS/logIOS.js]: nit: Please check that Xcode is installed.

**Comment by mkonicek** [comment]
Reviewed in the Tel Aviv hackathon.

@facebook-github-bot shipit

**Comment by facebook-github-bot-7** [comment]
Thanks for importing. If you are an FB employee go to [Phabricator](https://our.intern.facebook.com/intern/opensource/github/pull_request/1110713388995563/int_phab) to review.

**Pull request closed without merging**